ofctrl/fgraphSwitch.go

```diff
@@ -96,31 +97,39 @@ func (self *OFSwitch) DefaultTable() *Table {
 	return self.tableDb[0]
 }
 
-// Create a new group. return an error if it already exists
-func (self *OFSwitch) NewGroup(groupId uint32, groupType GroupType) (*Group, error) {
-	// check if the group already exists
-	if self.groupDb[groupId] != nil {
-		return nil, errors.New("group already exists")
+// NewGroup creates a new group; when using cache, returns an error if the group ID already exists, otherwise returns the
```

Comment:
```suggestion
// NewGroup creates a new group; when useCache is true, an error is returned if the group ID already exists, otherwise returns the
```